test/integration/preload_test.go

```diff
@@ -38,8 +38,7 @@ func TestPreload(t *testing.T) {
 
 	startArgs := []string{"start", "-p", profile, "--memory=3072", "--alsologtostderr", "--wait=true", "--preload=false"}
 	startArgs = append(startArgs, StartArgs()...)
-	k8sVersion := "v1.24.4"
-	startArgs = append(startArgs, fmt.Sprintf("--kubernetes-version=%s", k8sVersion))
+	startArgs = append(startArgs, fmt.Sprintf("--kubernetes-version=%s", legacyVersion()))
```

Comment:
I think this one is a test that we specifically do NOT have a preload for this kubernetes version and we are testing that no preload for this...this might break the test

I might be wrong but I am feeling we chose ""v1.24.4" as one that wont ever get a preload for testing purposes

Reply:
It may be possible, but we cannot do this. The tested version must be dynamic and change each time we update k8s version, on every release. If this test depend on not having a preload we need to find another way to cause to make preload lookup to fail, or delete the test.